melodic/distribution.yaml

```diff
@@ -14811,6 +14811,16 @@ repositories:
       url: https://github.com/anybotics/variant-release.git
       version: 0.1.6-1
     status: maintained
+  vda5050_connector:
+    doc:
+      type: git
+      url: https://github.com/tum-fml/vda5050_connector.git
+      version: v0.0.3
```

Comment:
The version listed here needs to be a branch, not a tag.  CI will fail if this is not a branch.  Same goes for all of the instances below.